terraform_files/main.tf

```diff
@@ -77,14 +78,16 @@ resource "libvirt_domain" "master" {
   }
 
   network_interface {
-    network_name = var.libvirt_network_name
+    network_name = libvirt_network.net.name
     hostname   = "${var.cluster_name}-master-${count.index}.${var.cluster_domain}"
-    addresses  = [var.libvirt_master_ips[count.index]]
+    addresses  = var.libvirt_master_ips[count.index]
+    wait_for_lease = true
```

Comment:
and if we don't start domain? only creating it? How it will work with wait_for_lease?

Reply:
Do we do that? @ori-amizur can you answer @tsorya's question?

Reply:
wait_for_lease is not mandatory.  OK to remove